packages/documentation/src/pages/platform/tools/feature-flags.mdx

```diff
@@ -1,33 +1,214 @@
 
-## Enhancements to Existing Applications
+Feature toggles:
+- Allow for production toggle switching without redeploying vets-website
+- Provide a user interface for managing feature toggle behavior
+- Provide code helpers for handling common user experience scenarios
+- Are powered by an open-source gem called [Flipper gem](https://github.com/jnunemaker/flipper)
 
-**STAGING**: Unreleased features should always be made available on [staging.va.gov](https://staging.va.gov) using [!isProduction()](https://github.com/department-of-veterans-affairs/vets-website/blob/10380007a3ec77047be10e4fb2a42d343c22c5d3/src/platform/utilities/environment/index.js)
+See the following sections for information about creating feature toggles:
+- [Writing good feature toggles](#writing-good-feature-flags)
+- Adding a new feature toggle
+   - [Frontend](#add-feature) 
+   - [Backend](#backend)
+- [Enabling and disabling features](#flipper-ui)
 
-**PRODUCTION**: Unreleased features can be made available on www.va.gov by checking [localstorage](https://github.com/department-of-veterans-affairs/vets-website/blob/92811be079cac99884abc28c2c96da61f08fe2a5/src/platform/utilities/storage/localStorage.js) for a developer-defined name/value
-   - Turning on the enhancement can be done by adding this name/value to the local storage
-      - With a console command ran by the user in the browser, or
-      - With application specific code that automatically checks when a certain query parameter is present.
+## <a id="writing-good-feature-flags">Writing good feature toggles</a>
 
-## New Applications
+Keep the following items in mind as you add feature toggles:
+- Remember that each environment has its own set of feature toggle values.
+- Test your feature toggle in staging before using it in production.
+- Remove feature toggles as soon as they are not needed.
+- Make toggles that are [easy to delete](https://programmingisterrible.com/post/139222674273/write-code-that-is-easy-to-delete-not-easy-to) by gating a behavior in as few places as possible. It's often better to have blocks of repeating code that can be quickly deleted later than it is to gate several small pieces of code.
+
+After a page is rendered, the feature toggle client retrieves the latest toggle values from the feature toggle service and the page is updated using the latest feature toggle values. The application shows a loading state for the new feature while the toggle values are retrieved from the service.
+
+## <a id="add-feature">Adding a new feature toggle (frontend)</a>
+
+Follow these steps to add and use a new feature toggle in vets-website:
+
+1. Determine your feature toggle name.
+
+   <b>Note:</b> There are no naming conventions yet. Current examples put the application name first, such as _facilityLocatorShowCommunityCares_ and _profileShowDirectDeposit_.
```

Comment:
FYI, you can wrap text in double asterisks to make it bold. 

For example:
```
**Note: **
```

Source: https://guides.github.com/features/mastering-markdown/#examples